src/http-exceptions/MethodNotAllowedException.php

```diff
@@ -11,4 +11,16 @@
 namespace Facebook\HackRouter;
 
 class MethodNotAllowedException extends HttpException {
+  public function __construct(
+    protected vec<HttpMethod> $allowed,
+    string $message = '',
+    int $code = 0,
+    ?\Exception $previous = null,
```

Comment:
Would Throwable be a better fit?

Reply:
we are passing this to `\Exception::__construct()`, `Throwable` won't work :/